vcpkg/concepts/ports.md

```diff
@@ -0,0 +1,30 @@
+---
+# concepts/ports
+title: Ports
+description: Ports as a vcpkg concept
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 09/21/2023
+ms.prod: vcpkg
+ms.topic: concept-article
+---
+
+# Concept: Ports
+
+In the context of the vcpkg package management system, a port is a predefined set of instructions and scripts that facilitate the easy integration of a specific library or package into C++ projects. A port comprises a collection of files that dictate how a library is built and installed, thus acting as a blueprint for managing library dependencies in a C++ environment.
```

Comment:
```suggestion
A port is vcpkg's method of delivering packages, at the very least a port contains:

* Metadata about the package: name, version, supported features, dependencies, etc.
* Instructions to acquire, build if necessary, and install the package. 

Additionally, ports often contain:

* Patches to apply to the package's source code
* Instructions to integrate with your project's build system
```

The introductory paragraph feels a bit redundant; I suggest we use a more direct description of what ports are.